src/cli.js

```diff
@@ -62,6 +62,13 @@ const {argv} = yargs
       'the root directory is found.\n',
     type: 'string',
   })
+  .option('p', {
+    alias: 'port',
+    describe: 'Port number to communicate via socket.\n' +
+      'A port number to talk to the socket established from IDE client.\n' +
+      'Required if the client communicates via socket connection.\n',
+    type: 'number',
+  })
```

Comment:
Don't forget to update the usage string above as well.

I find the wording here a little confusing, as it is not super clear whether this is the port number of a service running inside the IDE that the language service should connect to, or the opposite (that it is a port number that the language service itself should listen on). From my reading of the code, I understand it's the former.